File: java_wow_ironman_gen/WowClass.java
package java_wow_ironman_gen;

// No death knights.
// warrior, paladin, hunter, shaman, druid, rogue, monk, mage, warlock, priest
public enum WowClass {
    WARRIOR,
    PALADIN,
    HUNTER,
    SHAMAN,
    DRUID,
    ROGUE,
    MONK,
    MAGE,
    WARLOCK,
    PRIEST;

    public String toString() {
        return super.toString().toLowerCase();
    }
}

File: csharp/WowIronmanGen/WowIronmanGen/WowClass.cs
namespace WowIronmanGen;

public enum WowClass
{
    Warrior,
    Paladin,
    Hunter,
    Shaman,
    Druid,
    Rogue,
    Monk,
    Mage,
    Warlock,
    Priest
}

public static class WowClassExtensions
{
    public static string ToDisplayString(this WowClass wowClass)
    {
        return wowClass.ToString().ToLower();
    }
}
